content/en/security/threats/investigate_agent_events.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Investigate Agent Events
+kind: documentation
+disable_toc: false
+further_reading:
+  - link: "/security/default_rules/?category=cat-csm-threats#all"
+    tag: "Documentation"
+    text: "Explore CSM Threats detection rules"
+  - link: "/security/threats/workload_security_rules"
+    tag: "Documentation"
+    text: "Learn how to manage CSM Threats detection rules"
+  - link: "/security/notifications/"
+    tag: "Documentation"
+    text: "Learn more about security notifications"
+  - link: "https://www.datadoghq.com/blog/datadog-csm-windows/"
+    tag: "Blog"
+    text: "Secure your Windows workloads with Datadog Cloud Security Management"
+---
+
+
+This topic explains how to use the Agent Events explorer to query and review the Datadog Agent threat detection events generated by the [out-of-the-box (OOTB) detection rules][12].
+
+The Datadog Agent evaluates system activity on the Agent host. When activity matches an Agent rule expression, the Agent generates a detection event and passes it to the Datadog backend.
+
+If an event matches an Agent detection rule *and* a backend Threat detection rule, a signal is created and displayed in [Signals][11] (`Agent detection rule + backend Threat detection rule = Signal`).
+
+With the [Agent Events explorer][13], you can investigate Agent Events separately from signals. You can review the host path where the event happened, and view the event's attributes, metrics, and processes. You can also review the Agent rule that generated the event and view triage and response instructions.
+
+## Proactively block threats with Active Protection
+
+By default, all OOTB Agent threat detection rules are enabled and actively monitoring for threats.
+
+[Active Protection][12] enables you to proactively block and terminate threats identified by the Datadog Agent threat detection rules.
+
+## View Agent events
+
+To view Agent events, go to the [Agent Events explorer][13].
+
+Agent events are queried and displayed using the standard explorer controls in the Datadog [Events explorer][14]. 
+
+
+## Investigate Agent events
```

Comment:
I try to avoid screenshots when possible, but I feel like it would've helped to have one in this section (it took me a few minutes to figure out where the Path w/process tree was located on the side panel - I thought it was on the Processes tab at first, partly because the event I was looking at only had one command in the process tree).

Reply:
added screenshot